packages/medusa/src/api/routes/store/products/list-types.ts

```diff
@@ -0,0 +1,60 @@
+import { ProductService } from "../../../../services"
```

Comment:
Removing the endpoint would introduce a breaking change in the API. If we wish to keep theses changes as a `patch` changelog, my recommendation would be to set `deprecated:true` on the OpenAPI Spec for this route and create a Linear ticket for it future removal.

If we don't mind the breaking change, then I'll mark this as a minor and proceed with removing the endpoint.

@olivermrbl waiting for your guidance before proceeding.

Reply:
I am not entirely sure why removing the endpoint would introduce a breaking change, as it is not currently part of our store-facing API. Can I get you to elaborate?

It would introduce an inconsistency in our API as we would have `/admin/products/types` but not `/store/products/types`. So we could add the `deprecated` flag to `/admin/products/types` when we add `/admin/product-types`.

Let me know what you think. Also, sorry if I am missing something.

Reply:
Ah yes, deprecate the `/admin` and not introduce the `/store`. I'll proceed with that. Thanks.

Reply:
Done, see 237d679d36fcb6608fe2bcc689309454b77a9cbc